Code/Mantid/Framework/CurveFitting/src/DynamicKuboToyabe.cpp

```diff
@@ -0,0 +1,170 @@
+//----------------------------------------------------------------------
+// Includes
+//----------------------------------------------------------------------
+#include "MantidCurveFitting/DynamicKuboToyabe.h"
+#include "MantidAPI/Jacobian.h"
+#include "MantidAPI/FunctionFactory.h"
+#include <vector>
+
+namespace Mantid
+{
+namespace CurveFitting
+{
+
+using namespace Kernel;
+using namespace API;
+
+DECLARE_FUNCTION(DynamicKuboToyabe)
+
+// ** MODIFY THIS **
+// Here specify/declare the parameters of your Fit Function
+// 
+// declareParameter takes three arguments:
+//
+//   1st: The name of the parameter
+//   2nd: The default (initial) value of the parameter
+//   3rd: A description of the parameter (optional)
+//
+void DynamicKuboToyabe::init()
+{
+  declareParameter("Asym",  0.2, "Amplitude at time 0");
+  declareParameter("Delta", 0.2, "Local field");
+  declareParameter("Field", 0.0, "External field");
+  declareParameter("Nu",    0.0, "Hopping rate");
+}
+
+// Static Zero Field Kubo Toyabe relaxation function
+double ZFKT (const double x, const double G){
+
+  const double q = G*G*x*x;
+  return (0.3333333333 + 0.6666666667*exp(-0.5*q)*(1-q));
+}
+
+// Static Non-Zero field Kubo Toyabe relaxation function
+//double HKT (const double x, const double G, const double F) 
+//{
+//  throw std::runtime_error("HKT not implemented yet");
+//}
+
+// Dynamic Kubo-Toyabe
+double getDKT (double t, double G, double v){
+
+  const int tsmax = 656; // Length of the time axis, 32 us of valid data
+  const double eps = 0.05; // Bin width for calculations
+  //  const int stk = 25; // Not used for the moment
```

Comment:
The only reason for keeping it is that the HKT function will be used in the future (trac issue #11086), once I know how to implement it (there are some details that need to be discussed with muon scientists). But it is true that it is not used now, so I will be removing it for this ticket.